app/src/pages/admin/projectEventsPage/eventsGrid/eventsGrid.jsx

```diff
@@ -0,0 +1,184 @@
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import { Grid } from 'components/main/grid';
+import { AbsRelTime } from 'components/main/absRelTime';
+import styles from './eventsGrid.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  timeCol: { id: 'EventsGrid.timeCol', defaultMessage: 'Time' },
+  userCol: { id: 'EventsGrid.userCol', defaultMessage: 'User' },
+  actionCol: { id: 'EventsGrid.actionCol', defaultMessage: 'Action' },
+  objectTypeCol: { id: 'EventsGrid.objectTypeCol', defaultMessage: 'Object Type' },
+  objectNameCol: { id: 'EventsGrid.objectNameCol', defaultMessage: 'Object Name' },
+  oldValueCol: { id: 'EventsGrid.oldValueCol', defaultMessage: 'Old Value' },
+  newValueCol: { id: 'EventsGrid.newValueCol', defaultMessage: 'New Value' },
+});
+
+const TimeColumn = ({ className, value }) => (
+  <div className={cx('time-col', className)}>
+    <AbsRelTime startTime={value.lastModifiedDate} />
+  </div>
+);
+TimeColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+TimeColumn.defaultProps = {
+  value: {},
+};
+
+const UserColumn = ({ className, value }) => (
+  <div className={cx('user-col', className)}>{value.userRef}</div>
+);
+UserColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+UserColumn.defaultProps = {
+  value: {},
+};
+
+const ActionColumn = ({ className, value }) => {
+  const formattedActionType = value.actionType.split(/(?=[A-Z])/).join(' ');
+  return <div className={cx('action-col', className)}>{formattedActionType}</div>;
+};
+ActionColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+ActionColumn.defaultProps = {
+  value: {},
+};
+
+const ObjectTypeColumn = ({ className, value }) => (
+  <div className={cx('object-type-col', className)}>{value.objectType}</div>
+);
+ObjectTypeColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+ObjectTypeColumn.defaultProps = {
+  value: {},
+};
+
+const ObjectNameColumn = ({ className, value }) => (
+  <div className={cx('object-name-col', className)}>{value.details.objectName}</div>
+);
+ObjectNameColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+ObjectNameColumn.defaultProps = {
+  value: {},
+};
+
+const OldValueColumn = ({ className, value }) => (
+  <div className={cx('old-value-col', className)}>
+    {value.details.history.map((item) => (
+      <React.Fragment key={item.toString()}>
+        <div className={cx('value-field')}>{item.field}:</div>
+        <div className={cx('old-value')}>{item.oldValue}</div>
+      </React.Fragment>
+    ))}
+  </div>
+);
+OldValueColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+  value: PropTypes.object,
+};
+OldValueColumn.defaultProps = {
+  value: {},
+};
+
+const NewValueColumn = ({ className, value }) => (
```

Comment:
Can you avoid duplicating this function in the [`OldValueColumn`](https://github.com/reportportal/service-ui/pull/1143/files#diff-ff6769c5845d825e3ac5407c1bba3906R78)? They are definitely similar. F.e. put some `customProps` into the column to define which type is it (`newValue` or `oldValue` key).